src/lib/components/Panel.svelte

```diff
@@ -1,8 +1,13 @@
 <script lang="ts">
 	export let verticalAlign: 'top' | 'center' | 'bottom' = 'center'
+	const alignClassNames = {
+		top: 'panel-top',
+		center: 'panel-center',
+		bottom: 'panel-bottom',
+	}
 </script>
 
-<div class="panel panel-{verticalAlign}" {...$$restProps}>
+<div class="panel {alignClassNames[verticalAlign]}" {...$$restProps}>
```

Comment:
I like it, but, strictly speaking, this is not necessary, because both of your strings are in the same component, so `panel-{verticalAlign}` did not introduce implicit coupling, so it is not as bad as that case in `TaskRow`. However, it did introduce some cognitive load, and `alignClassNames` approach is better.

But what I asked for, was to stick to BEM convention, with which this is modifer, and so class names should be something like `panel panel--center`.